tests/common/mac/ProjectTestHelpers.cs

```diff
@@ -167,6 +167,7 @@ public class UnifiedTestConfig
 			public UnifiedTestConfig (string tmpDir)
 			{
 				TmpDir = tmpDir;
+				DiagnosticMSBuild = true;
```

Comment:
Instead of this, we should be getting rid of the entire option I think, since it is never set to false?